test/integration/fixtures/02-forms.js

```diff
@@ -6,18 +6,25 @@ const forms = [ simple, withrepeat ];
 module.exports = async ({ Assignments, Forms, Projects, Roles }) => {
   const project = (await Projects.getById(1)).get();
   const { id: formview } = (await Roles.getBySystemName('formview')).get();
+
+  // Create the forms without Enketo IDs in order to maintain existing tests.
+  global.enketo.state = 'error';
+
   /* eslint-disable no-await-in-loop */
   for (const xml of forms) {
     const partial = await Form.fromXml(xml);
-    // Create the form without Enketo IDs in order to maintain existing tests.
-    global.enketo.state = 'error';
-    const { acteeId } = await Forms.createNew(partial, project, true);
+    const form = await Forms.createNew(partial, project);
+    await Forms.publish(form, true);
 
     // Delete the assignment of the formview actor created by Forms.createNew()
     // in order to maintain existing tests.
-    const [{ actorId }] = await Assignments.getByActeeAndRoleId(acteeId, formview);
+    const [{ actorId }] = await Assignments.getByActeeAndRoleId(form.acteeId, formview);
     await Assignments.revokeByActorId(actorId);
   }
   /* eslint-enable no-await-in-loop */
+
+  // Reset enketo call count and error state to not affect tests
+  global.enketo.callCount = 0;
+  global.enketo.state = undefined;
```

Comment:
This is making me think that the `reset` function that's defined in test/util/enketo.js should get added to `global.enketo`. Right now, I'm pretty sure that function is just used in test/integration/setup.js, but if it were added to `global.enketo`, it could be used in more places (here, as well as any test that wants to reset the mock partway through the test).

Reply:
I have one or two places it tests that I might like to call a reset() function like this, but i was not sure how to get it added to `global.enketo`.